_api_app/app/Http/routes.php

```diff
@@ -38,8 +38,8 @@
         $entryRenderService = new App\Sites\Sections\Entries\SectionEntryRenderService([
             'entry' => $entry,
             'section' => $section,
-            'siteSettings' => $siteSettingsDataService->get(),
-            'siteTemplateSettings' => $siteTemplateSettingsDataService->get(),
+            'siteSettings' => $siteSettingsDataService->getWithDefaults(),
```

Comment:
Let's keep all the methods for frontend named the same. Just as you started with `getState` methods. I think that is a good idea because the reason for this method is not to add the default values, but to prepare the data for front-end. The default values are necessary only for the frontend, and in the backend, we simply use the `get` method for saving or manipulating data. Also if there would be any other adjustments necessary for the frontend data, we would put them in this method.

So let's rename it and add a comment to this method and to the doc string of the service class, that the `getState` method is meant for the frontend state.